### PR TITLE
Add multiplayer and official contests to `Help centre/Account restrictions`

### PR DESCRIPTION
**Pull request by Iyarita**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Help_centre/Account_restrictions/en.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ When an account is restricted, it is unable to interact with the community until
 
 The following features are disabled for restricted accounts:
 
+- multiplayer games
```
Comment (peppy) [inline wiki/Help_centre/Account_restrictions/en.md:11]:
```suggestion
- ability to participate in multiplayer rooms
```

might read better.

Comment (Iyarita) [reply wiki/Help_centre/Account_restrictions/en.md:11]: Sounds good to me.

**Comment by Iyarita** [comment]
I just happened to think, as long as I'm proposing changes to the list of restricted abilities, I thought of something that might be added:

-the ability to enter official contests (might need better wording)

**Review by TicClick — APPROVED** [review]
looks good to me

**Review by peppy — APPROVED** [review]

**Pull request merged**